src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -82,114 +85,133 @@
        :accessibility-label :summary-from-label}
       (i18n/label :t/details)]
      [rn/view
-      {:style (style/details-container theme)}
-      [quo/data-item
-       {:container-style style/detail-item
-        :blur?           false
-        :description     :default
-        :icon-right?     false
-        :card?           false
-        :label           :none
-        :status          :default
-        :size            :small
-        :title           (i18n/label :t/est-time)
-        :subtitle        (i18n/label :t/time-in-mins {:minutes (str estimated-time-min)})}]
-      [quo/data-item
-       {:container-style style/detail-item
-        :blur?           false
-        :description     :default
-        :icon-right?     false
-        :card?           false
-        :label           :none
-        :status          :default
-        :size            :small
-        :title           (i18n/label :t/max-fees)
-        :subtitle        (i18n/label :t/amount-with-currency-symbol
-                                     {:amount (str max-fees)
-                                      :symbol currency-symbol})}]
-      [quo/data-item
-       {:container-style style/detail-item
-        :blur?           false
-        :description     :default
-        :icon-right?     false
-        :card?           false
-        :label           :none
-        :status          :default
-        :size            :small
-        :title           (i18n/label :t/user-gets {:name (utils/get-shortened-address to-address)})
-        :subtitle        (str amount " " (:symbol token))}]]]))
+      {:style (style/details-container (some? route) theme)}
+      (if route
```

Comment:
Can we move the check on `route` to a let var? this will help us reuse it in this case.